docs/rest-api.md

```diff
@@ -49,7 +49,7 @@ Example:
 
 ```text
 Header: key: shuffle2024
-Body: settings: {"test": "test"}, model: "tiny", file: "test.wav"
+Body: settings: {"test": "test"}, model: "large-v3", file: "test.wav"
```

Comment:
Is this change needed?

Reply:
If you start the server it runs with the large-v3 and the tiny model example request won't work, so I changed the example for it to immediately work, wasn't that an issue for you?